src/components/implementation/tests/unit_pingshmem/ping.c

```diff
@@ -0,0 +1,29 @@
+#include <cos_kernel_api.h>
+#include <cos_types.h>
+#include <cos_component.h>
+#include <cos_defkernel_api.h>
+#include <capmgr.h>
```

Comment:
I'm not sure that all of these includes are required. Can you try pruning some out?